### PR TITLE
Bugfix in FR translation on declined submissions number

### PR DESCRIPTION
**Pull request by Gallyan**
The variable name for the number of declined submissions was not correct. I also remove the comma between month and year, that we don't use in french.

### CODE REVIEW

File: locale/fr_CA/emails.po [hunk 1]
```diff
@@ -1240,12 +1238,12 @@ msgstr ""
...
 "{$name}, <br />\n"
 "<br />\n"
-"Le rapport d'activité éditoriale de votre revue pour {$month}, {$year} est "
+"Le rapport d'activité éditoriale de votre revue pour {$month} {$year} est "
 "maintenant disponible. Les principales statistiques pour ce mois sont "
 "affichées ci-dessous.<br />\n"
 "<ul>\n"
 "\t<li> Nouvelles soumissions pour ce mois : {$newSubmissions} </li>\n"
-"\t<li> Soumissions refusées pour ce mois : {$rebuildsubmissions} </li>\n"
+"\t<li> Soumissions refusées pour ce mois : {$declineddsubmissions} </li>\n"
```
Comment (asmecher) [inline locale/fr_CA/emails.po:1246]: @Gallyan, I think this is a typo -- it should be `declinedSubmissions`.

File: locale/fr_FR/emails.po [hunk 2]
```diff
@@ -1374,12 +1374,12 @@ msgstr ""
...
 "{$name}, <br />\n"
 "<br />\n"
-"Le rapport d'activité éditoriale de votre revue pour {$month}, {$year} est "
+"Le rapport d'activité éditoriale de votre revue pour {$month} {$year} est "
 "maintenant disponible. Les principales statistiques pour ce mois sont "
 "affichées ci-dessous.<br />\n"
 "<ul>\n"
 "\t<li> Nouvelles soumissions pour ce mois : {$newSubmissions} </li>\n"
-"\t<li> Soumissions refusées pour ce mois : {$rebuildsubmissions} </li>\n"
+"\t<li> Soumissions refusées pour ce mois : {$declinedsubmissions} </li>\n"
```
Comment (asmecher) [inline locale/fr_FR/emails.po:1382]: I think this should be `declinedSubmissions` (note capitalization).

Comment (Gallyan) [reply locale/fr_FR/emails.po]: You're totally right

**Comment by asmecher** [comment]
Thanks, @Gallyan!

**Pull request merged**